src/services/config/geoNetworkClient.ts

```diff
@@ -0,0 +1,21 @@
+import axios, { AxiosError } from 'axios';
+import { apiBaseURL } from '../../utils/constants';
+
+const geoNetworkClient = axios.create({
+  baseURL: apiBaseURL,
+  headers: {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+  },
+});
+
+geoNetworkClient.interceptors.response.use(undefined, (error: AxiosError) => {
```

Comment:
can you also intercept the success and handle the message notification for accessibility?

Reply:
The request and response interceptor has been updated now.